src/librawspeed/decompressors/CrxDecompressor.cpp

```diff
@@ -238,7 +238,7 @@ inline int crxBitstreamGetZeros(CrxBitstream* bitStrm) {
   int32_t result = 0;
 
   if (bitStrm->bitData) {
-    _BitScanReverse((DWORD*)&nonZeroBit, (DWORD)bitStrm->bitData);
+    _BitScanReverse(static_cast<DWORD*>(&nonZeroBit), static_cast<DWORD>(bitStrm->bitData));
```

Comment:
@johnny-bit You have replaced DWORD with uint32_t at line 254, can we do the same here?